lib/greet.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+
+module.exports = exports = {};
+
+exports.sayHi = function greet(name) {
+  if(arguments.length === 0) throw new Error('name not provided');
+  return `hello ${name}!`;
+};
+
+exports.sayBye = function () {
+  console.log('adios');
+
+exports.randomGreeting = function(){
+  let someName = process.argv[2];
```

Comment:
You are grabbing the correct argument here and setting it to a variable.  To make this work though you then need to use that variable as a parameter where you call your sayHi function.